client/types/callbacks.go

```diff
@@ -39,6 +39,8 @@ type MessageData struct {
 }
 
 type Callbacks interface {
+	// OnConnect is called when the connection is successfully established to the Server.
+	// May be called after Start() is called and every time connected to the Server.
```

Comment:
```suggestion
	// May be called after Start() is called and every time a connection is established to the Server.
```

Reply:
Should we add more description here for websocket and http?

For example:

WebSocket clients will call OnConnect when a new WebSocket connection is established.

HTTP clients will call OnConnect after any request that results in a 200 response.

Reply:
Good suggestion. I will update the description.